src/message_protocols.ts

```diff
@@ -0,0 +1,7 @@
+// This file re-exports types for message protocols for background, content and commandline contexts
+
+export { Messages as Background } from "@src/background"
+export { Messages as CmdlineFrame } from "@src/commandline_frame"
+export { Messages as Content } from "@src/content"
```

Comment:
I don't like the fact that content script registers 3 different handlers because of ordering requirements, but I will try to defragment this later